tests/gui/modules/child-writer.spec.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const m = require('mochainon')
+const ipc = require('node-ipc')
+// eslint-disable-next-line no-unused-vars
+const childWriter = require('../../../lib/gui/modules/child-writer')
```

Comment:
Why are we requiring something we don't use? I think you can keep the linter happy by doing:

```js
require('../../../lib/gui/modules/child-writer')
```

Notice that by running the child writer, then we're running code that tries to connect to an IPC server that doesn't exist. Are no errors throws?

Reply:
Yup, forgot about that – fixed now! Yeah, there are no errors. `IPC_SERVER_ID` is `undefined` in `ipc.connectTo(IPC_SERVER_ID, () => { ...` which just exits silently as long as the verbosity is set to silent <sup>[1] [2]</sup>.

[1]: https://github.com/RIAEvangelist/node-ipc/blob/master/services/IPC.js#L226
[2]: https://github.com/RIAEvangelist/node-ipc/blob/master/services/IPC.js#L64